pinot-common/src/main/java/org/apache/pinot/common/metrics/ValidationMetrics.java

```diff
@@ -246,4 +247,13 @@ public void unregisterAllMetrics() {
     _metricNames.clear();
     _gaugeValues.clear();
   }
+
+  @VisibleForTesting
+  public long getValueOfGuage(final String fullGuageName) {
```

Comment:
typo s/guage/gauge in method name and also in parameter name

Reply:
thanks for the catch. I fixed this.